backend/core/api/views.py

```diff
@@ -23,9 +20,6 @@ def upload_file(request):
         true_df, pred_df = convert_file.convert(file_path, protected_attributes)
         results = calculate_metrics.calculate(true_df, pred_df, protected_attributes)
 
-        if os.path.exists(file_path):
```

Comment:
nitpick: I don't think we should delete this because it cleans out the files stored and the @csrf_exempt was put there to ignore some warnings - not sure @dhairya-t ?